images/kcp-registrar/register.sh

```diff
@@ -26,6 +26,7 @@ usage() {
     printf "The following parameters need to be passed to the script:\n"
     printf "KCP_ORG: the organization for which the workload clusters need to be registered, i.e.: root:pipelines-service\n"
     printf "KCP_WORKSPACE: the name of the workspace where the workload clusters get registered (created if it does not exist), i.e: compute\n"
+    printf "KCP_TAG: the tag of the kcp syncer image to use (preset in the image when using Pipelines as Code)\n"
```

Comment:
wdym its preset when using PAC? Do you mean that its set as an env var [here](https://github.com/openshift-pipelines/pipelines-service/blob/main/gitops/sre/.tekton/kcp-registration.yaml#L48)?

Reply:
No, it is set directly in the image (cf above). I added this comment as I thought it was not transparent.